creature.cpp

```diff
@@ -3,14 +3,18 @@
 Creature::Creature(std::string name, double life, double damage):
     name(name), life(life), damage(damage) {}
 
+Creature::Creature(const Creature& param):
+    name(param.getName()), life(param.getDamage()), damage(param.getDamage()) {}
+
+
 void Creature::attack(Creature* uj) const {
     if(uj->getLife() > 0){
         uj->setLife(uj->getLife() - this->getDamage());
     }
 }
 
```

Comment:
Ezt a függvényt Setter nélkül kell megvalósítani. Eleve a setter nélkül kellett már az első feladatot is.